content/en/news/releases/1.17.x/announcing-1.17.2/index.md

```diff
@@ -67,7 +66,7 @@ Crash when a redirect url without a state parameter is received in the OAuth fil
 
 - **Fixed** an issue causing VMs using auto-registration to ignore labels other than those defined in a `WorkloadGroup`.
   ([Issue #32210](https://github.com/istio/istio/issues/32210))
```

Comment:
This issue does not seem related at all to this particular text. Do we need another bullet?

Reply:
Text in 32210 release note:
```
**Fixed** reconciliation logic in the validation webhook controller to rate-limit
  the retries in the loop. This should drastically reduce churn (and generated logs)
  in cases of misconfiguration.
```

Reply:
Added, will sync the zh version soon.

Reply:
All Done